### ISSUE TITLE
Rename distribution

### ISSUE THREAD

**Issue by predic8** [membrane/service-proxy#424]
from membrane-service-proxy maybe to membrane-api-gateway

**Comment by predic8** [comment]
or even just to membrane-*

**Comment by rrayst** [comment]
The script files `service-proxy.bat`/`service-proxy.sh` are not renamed (yet?).

**Comment by predic8** [comment]
Yes. We can also rename them. I would like something short like run.sh. But I am not sure if we should rename those scripts rights now, cause we have to change a lot of documentation.
